backend/src/personnel/personnel.controller.ts

```diff
@@ -177,10 +178,29 @@ export class PersonnelController {
     @Param('id') id: string,
     @Req() req: RequestWithRoles,
     @Query() query: GetAvailabilityDTO,
-  ): Promise<AvailabilityEntity[]> {
+  ): Promise<AvailabilityRO[]> {
     this.logger.log(
       `${req.method}: ${req.url} - ${req.username}`,
     );
-    return await this.personnelService.getAvailability(id, query);
+    const dates = await this.personnelService.getAvailability(
+      id,
+      query,
+    );
+
+    const dateROs = dates.map((d) => d.toResponseObject());
+
+    const firstDate = dates[0];
+    if (firstDate.availabilityType !== 'NOT_INDICATED') {
```

Comment:
Will this always be true since NOT_INDICATED will not exist in the db?

Reply:
Oh good point

Reply:
Oh no, it won't. `getAvailability` as a function is the one that adds a bunch of NOT_INDICATEDs onto the array